src/lib/validators.ts

```diff
@@ -13,3 +13,19 @@ export const unixTimeFromDateString = new t.Type<number, string, unknown>(
     }),
   a => new Date(a).toISOString(),
 );
+
+export const validateEmail = (value: string | undefined | null | number) => {
```

Comment:
I'm not sure if we need to take so many different types into the function, I think only strings should be validated. If we have strongly typed app, then no undefineds or nulls should ever reach the function :p